nixos/modules/misc/ids.nix

```diff
@@ -276,6 +276,7 @@
       telegraf = 256;
       gitlab-runner = 257;
       postgrey = 258;
+      hound = 259;
```

Comment:
I'm really not sold on using preStart's to chmod a bunch of data. One of my hound installs has 700GB of owned data. I definitely don't want to be running chmod during a restart.